apps/tsconfig.json

```diff
@@ -11,9 +11,9 @@
     "jsx": "react",
     "target": "es5",
     "lib": ["ES2022", "DOM"],
-    "module": "es6",
+    "module": "node16",
```

Comment:
@sanchitmalhotra126 I tagged you on this PR because I changed some TypeScript configuration values.

Reply:
LGTM 👍 out of curiosity, looking at the tsconfig [docs](https://www.typescriptlang.org/tsconfig#moduleResolution) I see 
> These moduleResolution values, when combined with the corresponding [module](https://www.typescriptlang.org/tsconfig#module) values, picks the right algorithm for each resolution based on whether Node.js will see an import or require in the output JavaScript code.

Does this allow TS to resolve modules with `require` in our code now?

Reply:
I think maybe yes? But I'm not sure